### PR TITLE
addition of reconfirmation dialog box for logout issue #622 fixed

### PR DESCRIPTION
**Pull request by pranavgupta1234**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by pranavgupta1234** [comment]
@therajanmaurya can you please review this PR for #622

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java [hunk 1]
```diff
@@ -309,7 +311,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 setActionBarTitle(R.string.create_group);
                 break;
             case R.id.logout:
-                logout();
+                final AlertDialog.Builder builder = new AlertDialog.Builder(this);
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java]: @pranavgupta1234, Please use this [MaterialDialog](https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MaterialDialog.java) class.

**Comment by pranavgupta1234** [comment]
@therajanmaurya I had made the required changes :)

**Comment by tarun0** [comment]
@pranavgupta1234 Can you fix the conflicts? Please reply within a week if you can.

**Comment by pranavgupta1234** [comment]
@tarun0 Logout Reconfirmation Dialog box has already been added inside logout() function in MifosBaseActivity.java 
So I think we can close this issue.

**Comment by tarun0** [comment]
Closed as already solved.

**Pull request closed without merging**